src/App.vue

```diff
@@ -22,16 +23,35 @@ export default {
   },
   data() {
     return{
+      error:"",
       loggedIn: true,
       currentUser: {        
         "id": 1,
         "email": "hope.gochnour@gmail.com",
         "first_name": "",
         "last_name": "",
-        "language": "0",
-        "region": "0"
-        }
+        "language": "en-US",
+        "region": "US",
+        "watchlist": [],
+        "thumbs": [],
+        "recommendations": [],
+        "subscriptions": []
+        },
+      allGenres: [],
+      allServices: [],
+      allLanguages: [],
+      allRegions: []
     }
+  },
+    mounted() {
+      if(this.loggedIn) {
+        Promise.all([getGenres(this.currentUser.language), getRegions(this.currentUser.language), getServices(this.currentUser.region, this.currentUser.language)])
+        .then(responses => {
+          this.allGenres = responses[0],
+          this.allRegions = responses[1],
+          this.allServices = responses[2]
+        })
+      }
```

Comment:
Nice work getting the data to come through correctly!! Should be smooth sailing from here.